app/backend/wells/views.py

```diff
@@ -634,3 +636,35 @@ def lithology_geojson(request):
             get_env_variable('S3_WELL_EXPORT_BUCKET'),
```

Comment:
Please add env. variables to the openshift config: gwells/openshift/backend.dc.json

Reply:
done